Dnn.AdminExperience/ClientSide/Pages.Web/src/components/dnn-persona-bar-page-treeview/src/styles.less

```diff
@@ -123,7 +137,7 @@
     .selection-arrow {
         display: inline-block;
         position: relative;
-        width: 90px;
+        width: 50px;
```

Comment:
@valadas and others, this is the only bit that I see that might impact Platform only.

Long term, this Evoq only code shouldn't be here IMHO.

Reply:
I agree, we purposefully tried hard to gain as many pixels as possible for the very tight room we have in there. I would prefer if Evoq needs to shorten that, it is overriden in Evoq to not affect our efforts in this area for Platform.

@tauqeer-haider Would you be able to remove this change here and override it in Evoq css only ?